src/main/java/core/basesyntax/Calculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Calculator {
+    public double calculate(double operand1, double operand2, char operation);
```

Comment:
```suggestion
    double calculate(double operand1, double operand2, char operation);
```
у інтерфейся методи по дефолту є публічні і абстрактні